master/docs/developer/www-data-module.rst

```diff
@@ -10,12 +10,6 @@ Its main purpose is to handle the 3 way binding.
 2 way binding is the angular MVVM concept, which seamlessly synchronise the view and the model.
 Here, we introduce an additional way of synchronisation, which is from the server to the model.
 
-.. blockdiag::
-
-    blockdiag {
-      View <- Model <- Server;
-    }
-
```

Comment:
You did not replace this with an image?

Reply:
Too lazy as explained in the message of the first commit. Let's use svg from your PR here.

Reply:
:+1: